sdks/java/core/src/main/java/org/apache/beam/sdk/util/WindowedValue.java

```diff
@@ -671,7 +671,7 @@ public void encode(WindowedValue<T> windowedElem,
       Instant timestamp = InstantCoder.of().decode(inStream, nestedContext);
       Collection<? extends BoundedWindow> windows =
           windowsCoder.decode(inStream, nestedContext);
-      PaneInfo pane = PaneInfoCoder.INSTANCE.decode(inStream, nestedContext);
+      PaneInfo pane = PaneInfoCoder.INSTANCE.decode(inStream, context);
```

Comment:
Updated this coder to put value last, and fixed the `registerByteSizeObserver` that I missed the last time around.